oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java

```diff
@@ -510,10 +496,8 @@ private APIGatewayProxyResponseEvent handleAuthJourney(
                     authenticationRequest.getState(),
                     authenticationRequest.getResponseMode(),
                     OIDCError.UNMET_AUTHENTICATION_REQUIREMENTS,
-                    ipAddress,
-                    persistentSessionId,
                     authenticationRequest.getClientID().getValue(),
-                    clientSessionId);
+                    user);
         }
         var session = existingSession.orElseGet(sessionService::createSession);
         attachSessionIdToLogs(session);
```

Comment:
Would it be worth doing something like `user = user.withSessionId....` here instead of setting in the individual audit event? I'd be worried that we miss this off if we make changes in future